epochStart/mock/runTypeComponentsStub.go

```diff
@@ -3,20 +3,23 @@ package mock
 import (
 	"github.com/multiversx/mx-chain-go/process"
 	"github.com/multiversx/mx-chain-go/sharding"
+	"github.com/multiversx/mx-chain-go/sharding/nodesCoordinator"
 	"github.com/multiversx/mx-chain-go/testscommon"
 )
 
 // RunTypeComponentsStub -
 type RunTypeComponentsStub struct {
-	AdditionalStorageServiceFactory process.AdditionalStorageServiceCreator
-	ShardCoordinatorFactory         sharding.ShardCoordinatorFactory
+	AdditionalStorageServiceFactory  process.AdditionalStorageServiceCreator
```

Comment:
I think We can only use this one stub and remove the one from `process/mock`.
I checked in `baseProcess_test.go` to replace the same stub from `process/mock` with this one and it works;
This way we remove duplicated stubs

Reply:
done